content/faq.md

```diff
@@ -89,12 +87,17 @@ To put it another way: If you want to meet both WCAG 2.0 and WCAG 2.1, you can u
 
 WCAG 2.0 and WCAG 2.1 are both existing standards. WCAG 2.1 does not deprecate or supersede WCAG 2.0. W3C encourages you to use the most recent version of WCAG when developing or updating content or accessibility policies.
 
-## What's next after WCAG 2.1? {#next}
+## What's next for WCAG? What about “Silver”? {#next}
+
+The Accessibility Guidelines Working Group (AG WG) is developing WCAG 2.2 in 2020. The structure and content of WCAG 2.2 is the same as 2.1 and 2.0. Version 2.2 will include new accessibility requirements, called “success criteria”. You can review proposed additional success criteria in the [in-progress, unpublished “Editors’ Draft” of WCAG 2.3](https://w3c.github.io/wcag/guidelines/22/).
+
+AG WG is currently not planning to publish another version of WCAG 2, that is, not do a WCAG 2.3 — although that might change.
```

Comment:
Could this be softened a bit, e.g. "AG WG has not decided whether to publish another version of the WCAG 2, that is, whether to do a WCAG 2.3."

Reply:
Thank your for the more accurate info! I simplified it even more to:
> AG WG has not decided whether or not to do another version of WCAG 2 (WCAG 2.3).